File: src/main/java/de/cericx/coreapi/util/logger/Logger.java
package de.cericx.coreapi.util.logger;

import org.bukkit.Bukkit;

/***
 * @author Cericx_
 * @version 1.0
 * @since 11.07.2021
 */

public class Logger {

    public static void info(String... text) {
        for(String message : text) {
            Bukkit.getConsoleSender().sendMessage("§8[§9INFO§8] §7" + message);
        }
    }

    public static void error(String... text) {
        for(String message : text) {
            Bukkit.getConsoleSender().sendMessage("§8[§eWARN§8] §e" + message);
        }
    }

    public static void warn(String... text) {
        for(String message : text) {
            Bukkit.getConsoleSender().sendMessage("§8[§cERROR§8] §c" + message);
        }
    }

}

File: src/main/kotlin/de/cericx/coreapi/util/logger/Logger.kt
package de.cericx.coreapi.util.logger

import org.bukkit.Bukkit

/***
 * @author FlorianMichael as EnZaXD
 * @version 2.0
 * @since 11.07.2021
 */

object Logger {

    internal fun log(level: Level, vararg message: String) {
        for (s in message)
            Bukkit.getServer().consoleSender.sendMessage(level.display + " " + s)

    }

    fun info(vararg message: String) {
        log(Level.INFO, *message)
    }


    fun warn(vararg message: String) {
        log(Level.WARN, *message)
    }

    fun error(vararg message: String) {
        log(Level.ERROR, *message)
    }

    enum class Level(var display: String) {
        INFO("§7Info"),
        WARN("§eWarn"),
        ERROR("§cError");
    }
}
